### DISCUSSION TITLE
Assioma Uno not showing up in BLE?

### DISCUSSION THREAD

**Discussion by yetdog** [doudar/SmartSpin2k#520] [Q&A] [answered]
I saw a previous discussion about this but it was deemed fixed by a release back in January 2021. I'm running 23.12.21-develop and no matter what, I can't get the bluetooth scanner to show my assioma pedal. My HRM shows up when scanning just fine.

If I need to upgrade my firmware, I'd like to, but it asks me for a username/password when clicking on the update firmware link.

**Comment by doudar** [comment, marked as answer]
Typically it's because something else is grabbing the Assioma before SS2k does, so make sure to check your other devices aren't searching/or connected to it. Power down everything, start the SS2k, start pedaling (to turn on the pedals), and then turn on your hr. It should show up. 

If you do need to upload new firmware, the user and password is admin. Mac safari (in hotspot login mode) doesn't work. Chrome seems to work best. 

Thanks!

**Reply by yetdog** [reply]
Thanks for the fast comment! I’ll try those steps in that order shortly. 

One thing that’s odd…the Zwift app, if I tap the power meter, shows me both the SS2K and the Assioma (but I don’t connect to either) so I would think the pedal was still “unpaired”. I’ll shut that off as well and only bring it up after I get things sorted in SS2K. 

Will respond after testing. Thanks again!

**Reply by yetdog** [reply]
You mention turning on HR; is that a necessary. Step?

**Reply by yetdog** [reply]
That was it, thank you for the clarification!

**Reply by doudar** [reply]
Yes, it seems that whenever Zwift is "scanning" it actually claims the devices and makes them inaccessible to anything else. Now that you've got it all setup, it should "just work". Glad I could help!
